Week1/homework/app.js

```diff
@@ -1,11 +1,182 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+(function() {
+  const booksList = [
+    'home_and_world',
+    'the_rescue',
+    'heart_of_darkness',
+    'country_of_pointed_fires',
+    'howards_end',
+    'robinson_crusoe',
+    'the_odyssey',
+    'death_in_venice',
+    'adventures_of_huckleberry_finn',
+    'oliver_twist',
+    'frankenstein',
+    'return_of_native'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
-}
+  /*
+  function allBooks() {
+    const ul = document.createElement('ul');
+    for (let book of booksList) {
+      const li = document.createElement('li');
+      ul.appendChild(li);
+      li.setAttribute('id', book);
+      console.log(li);
+    }
+  }
+  console.log(allBooks());
+
+  */
+
+  const information = {
+    home_and_world: {
+      author: 'Rabindranath Tagore',
+      name: 'The Home and the World',
+      alt: 'This is the cover of the book called The Home and the World',
+      description:
+        "Originally written in Bengali and published in 1916, The Home and the World is the vision of Rabindranath Tagore, Asia's first Nobel laureate."
+    },
+    the_rescue: {
+      author: 'Joseph Conrad',
+      name: 'The Rescue',
```

Comment:
Something which Jim pointed out in the Slack channel. The requirements states: 

`1.4 Make an object (not an array!) containing information for each book. Each property of this object should be another (i.e., nested) object with the book ID you thought up in step 1.1 as a key, and at least the following properties: title, language and author.`

However, the properties which you are using are different except for `author`. It might seem very minor right now as you are developing alone but imagine a situation where you work with a Backend developer. You agree on a contract related to naming conventions but now that you use different names for properties, you are in a breach of that contract which would eventually lead to bugs in your application. Please make sure that following the requirements is priority number 1. 🙂

Reply:
yes sure. I will focus on this thing.
thank you very much ^_^